docs/index.html

```diff
@@ -88,7 +145,7 @@ <h4>
             </span>
           </p>
           <div class="cde" id='headerLinkCode'>
-            &lt;link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/devicons/devicon@{{ latestReleaseTagging }}/devicon.min.css"&gt;
+            &lt;link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/devicon.min.css"&gt;
```

Comment:
Any reason why we can't use `<` and  `>` instead of `&lt;` and `&gt;`?
Also, why did we change this to latest? Won't it cause issues when we want to test css for other releases than the latest?

Reply:
1. Not sure what the difference between those two is. I could try, but I'm not sure it will benefit anything.
2. Latest will always fetch the latest **released version** in NPM (in this case, 2.15.1), so it is the same as specifying the version manually or using `@latest`, which is faster, though. No errors should happen, I already tested it and it works fine.

Reply:
1. Should be the same (I think), but it looks weird with `&lt;` instead of `<`
2. Hmm, what if we made a breaking change, and we needed to see the difference between latest and say version 2.15.0. wouldnt it be easier if `@latest` wasn't hardcoded?

Reply:
> difference between latest and say version 2.15.0

What do you mean exactly? 🤔

Reply:
What if release 15.2.0 brought a breaking change.
To debug we might want to check out the 15.1.0 release. If `@latewt` is hardcoded like it is now, wouldn't we beed to change to `15.1.0` everywhere, instead of just one place (like before)?

Reply:
That's a good point. I tried hardcoding it like that before, but for some reason it didn't work.
I could try again if wanted, or you could try it yourself (let me know what works best for you).

Reply:
Feel free to try if you'd like, I'll also take a look ASAP (which is hopefully tomorrow)

Reply:
`&lt;` and `&gt;` are html escape sequences to literal characters `<` and `>`. So if we want to show a string with those characters on html (hardcoded) we need to use the escape sequences, otherwise the string will be interpreted as a code.
A workaround can be achieved by inserting this text via JavaScript, like:

***index.html***
```html
<div class="cde" id='headerLinkCode'></div>
```
***./assets/js/script.js***
```js
const headerLinkCodeElem = document.getElementById('headerLinkCode');
headerLinkCodeElem.textContent = `<link rel="stylesheet"href="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/devicon.min.css">`;
```
However, I'm ok as it is now. 😁

I agree with @Snailedlt on not hardcode `@latest`, for the same reasons.

Reply:
Just to make it absolutely clear. 
`&lt;` and `&gt;` are used here, because we want to show `<` and `>` inside these fields (which means we need to use the escape-sequence instead of the actual character symbol): 
![image](https://user-images.githubusercontent.com/43886029/227168700-b9c6d520-258d-41da-8bdf-bb9b427ebe66.png)

Reply:
I changed back to variable version, and moved the JS code to the .ja file in this PR: https://github.com/Panquesito7/devicon/pull/187

@lunatic-fox feel free to take a look as well. I wasn't able to get the local SVG's working on my and panquesito's branch, so if you could test it locally, that would be neat!

Reply:
Exactly! 🙂